src/scripts/OSFramework/OSUI/Pattern/Video/Video.ts

```diff
@@ -411,8 +413,14 @@ namespace OSFramework.OSUI.Patterns.Video {
 		 * @param {number} time value in seconds
 		 * @memberof Video
 		 */
-		public setVideoJumpToTime(time: number): void {
-			this._videoElement.currentTime = time;
+		public setVideoJumpToTime(time: string): void {
+			const jumpToTime = time.split(':');
+			// Convert the time provided into seconds
+			const timeInSeconds =
+				+jumpToTime[0] * Enum.VideoTime.Hour * Enum.VideoTime.Minute +
+				+jumpToTime[1] * Enum.VideoTime.Minute +
+				+jumpToTime[2];
+			this._videoElement.currentTime = timeInSeconds;
```

Comment:
@bmarcelino-fe Why do we need this to be string in here? Because of the Time type the platform generates?

If that so, I would say it would be better to do that before, before calling the API. We already in other locations I think, like a Helper to translate platform time to expected js time.

That way we wouldn't pollute the javascript API